docs/UserGuide.md

```diff
@@ -367,6 +400,48 @@ Examples:
 * `list_d` followed by `delete 2` deletes the 2nd doctor in the doctor list.
 * `find_d Jonathan` followed by `delete 1` deletes the 1st doctor in the results of the `find_d` command.
 
+### Adding a specialisation : `add_tag_d`
+
+Adds the specified specialisation from a doctor from the system.
+
+Format: `add_tag_d INDEX [t\SPECIALISATION]`
+
+* Adds the specialisation of doctor at the specified `INDEX`.
+* The index refers to the index number shown in the displayed doctor list.
+* The index **must be a positive integer** 1, 2, 3, …
+
+Examples:
+* `list_d` followed by `add_tag_d 2 Orthopaedic` add specialisation the 2nd doctor in the doctor list.
+* `find_d Jonathan` followed by `add_tag_d 1 Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
+
+### Deleting a specialisation : `delete_tag_d`
+
+Deletes the specified specialisation from a doctor from the system.
+
+Format: `delete_tag_d INDEX [t\SPECIALISATION]`
+
+* Adds the specialisation of doctor at the specified `INDEX`.
+* The index refers to the index number shown in the displayed doctor list.
+* The index **must be a positive integer** 1, 2, 3, …
+
+Examples:
+* `list_d` followed by `delete_tag_d 2 Orthopaedic` add specialisation the 2nd doctor in the doctor list.
+* `find_d Jonathan` followed by `delete_tag_d 1 Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
+
+### Editing a remark : `remark_d`
+
+Edits the remark of a doctor from the system.
```

Comment:
Can talk about what a remark can be used for. In our case we talk about indicating the doctor's availability.

Reply:
done in new update! ty